tests/regression.py

```diff
@@ -1,3 +1,4 @@
+#! python3
```

Comment:
This is incorrect on unix. "python3" is not an absolute path. It would have to be something like

```suggestion
#!/usr/bin/env python3
```

Reply:
Thanks for pointing that out. The GitHub Actions workflow does not seem to have a problem with it, but it's always better to be on the safe side.